src/commands/dev/dev.js

```diff
@@ -221,6 +228,8 @@ const startFrameworkServer = async function ({ settings }) {
     log(NETLIFYDEVERR, `Please make sure your framework server is running on port ${settings.frameworkPort}`)
     exit(1)
   }
+
+  return { ipVersion: port?.ipVersion }
```

Comment:
oooh, I like that we can finally use optional chaining in the CLI!

Reply:
yea because we dropped node 12 🥳